src/main/java/mate/academy/spring/dao/UserDaoImpl.java

```diff
@@ -6,12 +6,14 @@
 import org.hibernate.Session;
 import org.hibernate.SessionFactory;
 import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public class UserDaoImpl implements UserDao {
     private final SessionFactory sessionFactory;
 
+    @Autowired
```

Comment:
FYI
it's not necessary to use this annotation here, Spring will inject either way